src/test/java/org/prebid/server/auction/CurrencyConversionServiceTest.java

```diff
@@ -39,7 +39,7 @@
     private static final String EUR = "EUR";
     private static final String UAH = "UAH";
     private static final String AUD = "AUD";
-    private static final String URL = "http://currency.prebid.org/latest.json";
+    private static final String URL = "https://cdn.jsdelivr.net/gh/prebid/currency-file@1/latest.json";
```

Comment:
Pls, change to something unreal, like `http://currency-rates/latest.json`
Unit test doesn't need real url, so next time it is changed we won't change this line.